src/main/kotlin/racingcar/domain/Car.kt

```diff
@@ -0,0 +1,33 @@
+package racingcar.domain
+
+class Car(name: String) {
+    var position: Int = DEFAULT_POSITION
+        private set
+
+    var carName: CarName = CarName(name)
+        private set
+
+    fun move(condition: Int) {
+        if (condition >= MOVE_CONDITION) {
+            position++
+        }
+    }
+
+    companion object {
+        private const val DEFAULT_POSITION: Int = 0
+        private const val MOVE_CONDITION: Int = 4
+    }
+}
+
+class CarName(name: String) {
+    var name: String = name
+        private set
```

Comment:
name을 var로 따로 선언할 필요가 있을까요? 

private set로 선언했다는 것은 변경될 소지가 없다는 것이니 아래와 같이 선언해두어도 괜찮을것 같습니다.

```suggestion
class CarName(val name: String) {
```